src/plugins/size/README.md

```diff
@@ -0,0 +1,41 @@
+- infos = Information about the size plugin is in keys below
+- infos/author = Thomas Waser <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = conv
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained conformant compatible coverage specific unittest shelltest tested nodep libc configurable experimental
+- infos/metadata = transform/size
+- infos/description = transforms WIDTHxHEIGHT values into separate keys
+
+## Introduction
+
+The `size` plugin transforms values with the format `WIDTH<separator>HEIGHT` into a separate `Width` and `Height` key. The metakey `transform/size` triggers the conversion.
```

Comment:
Thanks! If rename is done earlier than size, different capitalizations of width and height would not work. Its only relevant if rename and this plugin use the same placements. If they do, a simple `infos/ordering = rename` in the contract of this plugin here should fix the potential problem.